hack/cherry_pick_pull.sh

```diff
@@ -44,8 +44,8 @@ if [[ -z ${GITHUB_USER:-} ]]; then
   exit 1
 fi
 
-if ! which hub > /dev/null; then
-  echo "Can't find 'hub' tool in PATH, please install from https://github.com/github/hub"
+if ! command -v gh > /dev/null; then
+  echo "Can't find 'gh' tool in PATH, please install from https://github.com/cli/cli"
```

Comment:
they have to login too right? before running this script ("gh auth login")?

Reply:
Added a check for this